### ISSUE TITLE
RFC: 'Column display options' -> 'Toggle all/none selected' wording

### ISSUE THREAD

**Issue by petecooper** [textpattern/textpattern#2057]
Admin side content list views each have a 'Column display options' setting, and all have a 'Toggle all/none selected' entry at the top of the list immediately after the 'Toggle all/none selected' header:

<img width="550" height="600" alt="Image" src="https://github.com/user-attachments/assets/40f76bcf-5358-4e35-8a62-1b579b2a9c64" />

The 'Toggle all/none selected' header selects or deselects (toggles) the items in the 'Column display options'. This makes sense. All good. The 'Toggle all/none selected' at the top of the list _after_ the header shows or hides the entire first column in the content list…not just the 'Toggle all/none selected' checkbox at the top of the column.

Proposal for comment: change the wording on the 'Toggle all/none selected'  line item to 'Select item(s)' or 'Select' or similar:

<img width="610" height="592" alt="Image" src="https://github.com/user-attachments/assets/ea6ba0a7-2a5c-4255-891a-b08c9783ba73" />

**Comment by Bloke** [comment]
Ummm, there shouldn't be a second option at all, let alone be named the same as the toggle at the top. Weird. Umm, @bloatware have I broken something?

**Comment by bloatware** [comment]
@Bloke who else? :-) It looks ok in non-content lists (sections etc). Have you any memories of specifically modifying the content ones? Probably, [bulk](https://github.com/textpattern/textpattern/commit/853623f705bee133f96a0367b25d5634a75a992d) and alike? Weird case, this widget looks too fragile.

**Comment by bloatware** [comment]
It looks like adding `<label for="select_all" />` has broken it.

**Comment by Bloke** [comment]
Dammit. That was probably an accessibility "improvement" that backfired. Can we work around it easily or do we need to rethink how it's built or how we add the accessibility necessities?

**Comment by bloatware** [comment]
I don't really understand what this improvement improves, all `select_all` checkboxes seem already be wrapped in a label. Does the latter need `for` attribute and `txp-accessibility` class?

**Comment by Bloke** [comment]
I don't think it was meant to target those. They were already marked up well enough. This was for the other elements that didn't already have it, but it got carried away and applied it here too. At least, that's my best guess. It was a while ago.

**Comment by bloatware** [comment]
That's what I get when removing the extra label:
``` html
<li class="txp-dropdown-toggle-all ui-menu-item">
  <div role="menuitem" id="ui-id-2" tabindex="-1" class="ui-menu-item-wrapper">
    <label>
      <input tabindex="-1" class="checkbox active" data-name="select_all" type="checkbox" checked="checked">
      Toggle all/none selected
    </label>
  </div>
</li>
```
Is it ok?

**Comment by phiw13** [comment]
The cause of that duplicate item in the column display widget comes from the first column (checkboxes) in list table (if you toggle that second checkbox, the column appears or disappears). If you change the “toggle-all” checkbox (in the `th`) to use `aria-label="gTxt('toggle_all_selected')"` instead of the current `<label for………>` that will fix that column display widget. For some reason beyond and above my pay grade,  the JS script that builds that menu now thinks the checkboxes column should now become toggable.

Expected markup

```
<th class="txp-list-col-multi-edit" scope="col">
    <input aria-label="Toggle all/none selected" name="select_all" id="select_all" type="checkbox" value="0">
</th>
```

After that the column display menu reverts to previous incarnation, no duplicate item, same as e.g  the Sections panel.

Possible?

Affected panels: Articles, Images, Files, Links, btw.

**Comment by bloatware** [comment]
Nice observation, thank you. I have patched article lists atm, please test.

**Comment by phiw13** [comment]
Looks good. V.O can hear it, accessibility checkers are still happy and the “column display options” menu reverted to previous state as expected.

**Comment by bloatware** [comment]
Thank you, should be fixed now in all list panels.

**Comment by petecooper** [comment]
Thank you @bloatware - greatly appreciated.
